.dockerignore

```diff
@@ -8,8 +8,8 @@ build/**/*
 build
 dist/**/*
 dist
-fmriprep.egg-info/**/*
-fmriprep.egg-info
+dmriprep.egg-info/**/*
```

Comment:
Can you submit these infrastructure changes on a different PR? I'll get them merged quickly